src/Parse.ts

```diff
@@ -0,0 +1,449 @@
+import decode from './decode';
```

Comment:
@dblythy Since you deleted the old Parse.js and created Parse.ts instead of renaming wouldn't this delete the commit history?